skyportal/handlers/api/source.py

```diff
@@ -439,6 +441,11 @@ def post(self):
                 "Invalid group_ids field. Please specify at least "
                 "one valid group ID that you belong to."
             )
+
+        # use ra, dec as the discovery position unless it's already specified
+        obj.ra_dis = obj.ra if obj.ra_dis is None else obj.ra_dis
```

Comment:
How do you know `obj.ra` always correspond to the discovery ra? And vice versa for dec?

Reply:
Again, it's just to get the *_dis variables loaded at save time.